src/lib/utils/statistics-helper.ts

```diff
@@ -0,0 +1,29 @@
+import { type Paper, ReviewDecision } from "$lib/model/backend";
+
+/**
+ * Calculate the progress of a stage.
+ *
+ * The progress is defined as the number of papers that are already accepted or declined (so no further
+ * decision need to be made) relative to the total number of papers (including the unreviewed a paper that need
+ * the decision of the arbiter).
+ *
+ * If no papers are provided, the progress is set to 0 per default.
```

Comment:
Minor typos
```suggestion
 * Calculate the progress of a stage.
 *
 * The progress is defined as the number of papers that are already accepted or declined (so no further
 * decision is required) relative to the total number of papers (including the unreviewed papers that need
 * the decision of the arbiter).
 *
 * If no papers are provided, the progress is set to 0 by default.
```

Shouldn't it be "including the reviewed papers that need the final decision of the arbiter"?